R/zchunk_L232.water.demand.manufacturing.R

```diff
@@ -32,41 +31,32 @@ module_water_L232.water.demand.manufacturing_DISABLED <- function(command, ...)
     A32.globaltech_coef <- get_data(all_data, "energy/A32.globaltech_coef")
     L132.water_coef_manufacturing_R_W_m3_GJ <- get_data(all_data, "L132.water_coef_manufacturing_R_W_m3_GJ")
 
-    # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are `merge` calls in this code. Be careful!
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: there are 'match' calls in this code. You probably want to use left_join_error_no_match
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # ===================================================
+    L132.water_coef_manufacturing_R_W_m3_GJ %>%
+      mutate(supplysector = dplyr::first(A32.globaltech_coef$supplysector),
```

Comment:
it seems to be necessary... it's because `data.table` has a `first` function also I think... actually I'm thinking I can just import `first` at the head of the chunk and it will probably work.